src/Presentation/Nop.Web/Views/Shared/_ProductBox.cshtml

```diff
@@ -24,9 +24,38 @@
 }
 <div class="product-item" data-productid="@Model.Id">
     <div class="picture">
-        <a href="@Url.RouteUrl("Product", new { SeName = Model.SeName })" title="@Model.DefaultPictureModel.Title">
-            <img alt="@Model.DefaultPictureModel.AlternateText" src="@Model.DefaultPictureModel.ImageUrl" title="@Model.DefaultPictureModel.Title" />
-        </a>
+        @if (Model.PictureModels.Count > 1)
+        {
+            var rtl = (await workContext.GetWorkingLanguageAsync()).Rtl;
```

Comment:
Use `ShouldUseRtlThemeAsync` method to define whether to use RTL in the frontend. See a sample [here](https://github.com/nopSolutions/nopCommerce/blob/master/src/Presentation/Nop.Web/Views/Shared/_Root.Head.cshtml#L44).